R/dataprep_table_formatting.R

```diff
@@ -43,7 +43,7 @@ format_as_update_existing <- function(sidora_table) {
   # as displayed (particularly to prevent datetime saving with TZ info)
   result %>% 
     dplyr::rename_with(make_column_mandatory, valid_cols_table_clean_mandatory) %>%
-    dplyr::mutate(dplyr::across(tidyselect::everything(), as.character))
+    dplyr::mutate(dplyr::across(tidyselect::everything(), format, trim = TRUE))
```

Comment:
I'm not entirely sure if that is stable across all data types

Reply:
Can try and wait for the explosions